models/classes/class.KeyValueResultStorage.php

```diff
@@ -350,7 +350,7 @@ private function subStrPrefix(&$value, $key, $prefix)
     public function getVariableProperty($variableId, $property)
     {
         $variableIds = explode('http://',$variableId);
-        $variableId = "http://".$variableIds[2];
+        $variableId = $variableIds[0] . 'http://' . $variableIds[2];
```

Comment:
It may sound a bit more but can you move this logic to a method like:
And then just call the method here ?
```php
    /**
     * @param string $variableId
     * @return string
     */
    protected function extractDeliveryExecutionId($variableId)
    {
        $parts = explode('http://',$variableId);
        if (isset($parts[2])) {
            return $parts[0] . 'http://' . $parts[2];
        }

        return $variableId;
    }
```